### DISCUSSION TITLE
How to turn it on - userChrome.bookmarkbar.multi_row?

### DISCUSSION THREAD

**Discussion by Someone-G** [black7375/Firefox-UI-Fix#830] [Q&A]
How can I enable this feature?
> userChrome.bookmarkbar.multi_row

**Comment by black7375** [comment]
I found that it didn't work on the latest Firefox, and fixed it.
- https://github.com/black7375/Firefox-UI-Fix/commit/431d32e29e58557a71b480f43cde10e371d012ec

You can enable option with `about:config`
- https://github.com/black7375/Firefox-UI-Fix/wiki/Tutorial#set-using-aboutconfig
